app/src/main/res/values/strings.xml

```diff
@@ -34,7 +34,7 @@
     <string name="invalidLonTitle">Invalid longitude</string>
     <string name="invalidLonMsg">The longitude must be between -180.0° and +180.0°.</string>
     <string name="invalidDoubleTitle">Invalid input</string>
-    <string name="invalidDoubleMsg">Failed to read number. Expected the format 25.314.</string>
+    <string name="invalidDoubleMsg">Failed to read number. Expected the format "25.314".</string>
```

Comment:
Doesn´t really matter but these quotation marks are not shown in the toast.

Generally I would suggest using non-random numbers for placeholders like "12.345" to direct attention to the format, not the numbers.